### DISCUSSION TITLE
Spring equinox

### DISCUSSION THREAD

**Discussion by Martin-Reuter** [Stellarium/stellarium#3779] [Q&A] [answered]
Hello,
the spring equinox is the day on which the length of the day is equal to the length of the night, i.e. 12 hours in each case.
Stellarium gives a day length of 12 hours (when the sun is observed) on March 17, 2024, while officially March 21, 2024 is given as the beginning of spring (spring equinox).
What is the mistake?

**Comment by 10110111** [comment, marked as answer]
> the spring equinox is the day on which the length of the day is equal to the length of the night

Not really. The equinox is defined by the position of the Sun relative to the Earth's equator. Due to the Sun being about 0.5° tall and the effects of atmospheric refraction this doesn't let the duration of the day equal that of night at the day of equinox (they are defined by the moments of (dis)appearance of the edge of the solar disk with respect to the horizon). [Wikipedia](https://en.wikipedia.org/wiki/Equinox#Length_of_equinoctial_day_and_night) has some discussion of this.

**Reply by gzotti** [reply]
In addition, the modern geometic definition is just that Sun has reached geocentric apparent ecliptical longitude of 0°. See discussion in Meeus, Morsels II, ch. 72.

EDIT: Sorry, spring equinox is of course 0°. Just answering at summer solstice/90°...

**Reply by Martin-Reuter** [reply]
Thank you for good explanation.
Which measurement value for the sun do I have to pay attention to in Stellarium in order to recognize when the spring equinox is?

**Reply by gzotti** [reply]
> that Sun has reached geocentric apparent ecliptical longitude [of date] of 0°.

**Reply by gzotti** [reply]
BTW, "officially" it is March 20 for the rest of this century IIRC. At least in Europe (Your timezone may vary...).

**Reply by Martin-Reuter** [reply]
![Bild_2024-06-23_205145393](https://github.com/Stellarium/stellarium/assets/173625137/6d5c8e03-4b2a-48e9-837b-14672d6cac96)
Is it this?

**Reply by gzotti** [reply]
If you now make sure to have switched off Topocentric correction and adjust time so that `Ecl.long/lat (on date)` has 0°0'0.0" as first value, yes. (SPOILER: My system shows it to have been at 4:06:21 CET.)

**Reply by Martin-Reuter** [reply]
Where can I switch off Topocentric correction?

**Reply by gzotti** [reply]
For such questions I point you to our User Guide which is on your harddisk and linked in your Start menu. No need to print all 474 pages and read from end to end. It is a PDF and can be searched. (Hint: just search "topocentr" to avoid missing spelling variants)

**Reply by Martin-Reuter** [reply]
Thank you.
Is it this?
![Bild_2024-06-24_122156261](https://github.com/Stellarium/stellarium/assets/173625137/1095f75f-9490-4572-a062-3fb80b946702)
